src/app/app-models/components/add-model/add-model.component.ts

```diff
@@ -199,5 +208,14 @@ export class AddModelComponent implements OnInit, OnDestroy {
     this.addedModel = false;
     this.loading = false;
   }
+
+  getModelName(id: string, models: types.DeCaF.ModelHeader[]): string {
```

Comment:
This works, but something similar could be necessary in other components too. Would it be possible with a selector or similar that picks the model object based on an id? So that it's not specific to this component, and not specific to only the `name` attribute.

Reply:
You are totally right, I didnt think about it, but it is better do it like a shared function, because I am using the same in other components.